packages/typegpu/src/data/struct.ts

```diff
@@ -121,13 +127,16 @@ class TgpuStructImpl<TProps extends Record<string, AnyTgpuData>>
   }
 }
 
-function getAttribute(field: AnyTgpuData): string | undefined {
-  if (field instanceof TgpuAlignedImpl) {
+function getAttribute<T extends AnyTgpuData>(field: T): string | undefined {
+  if (isAlignedSchema(field as unknown)) {
     return `@align(${field.byteAlignment}) `;
   }
-  if (field instanceof TgpuSizedImpl) {
+
+  if (isSizedSchema(field as unknown)) {
     return `@size(${field.size}) `;
   }
+
+  return undefined;
```

Comment:
this line is unnecessary, was it added to make the undefined return more explicit?

Reply:
Yess 🙌